### PR TITLE
Cleanup epoch slots

### PR DESCRIPTION
**Pull request by aeyakovenko**
#### Problem

We only use the lowest slot form EpochSlots.  

#### Summary of Changes
Clean up the type, rename it to LowestSlot and make room for EpochSlots in #8552 

Fixes #

### CODE REVIEW

File: core/src/crds_value.rs [hunk 1]
```diff
@@ -61,7 +59,7 @@ impl Signable for CrdsValue {
 pub enum CrdsData {
     ContactInfo(ContactInfo),
     Vote(VoteIndex, Vote),
-    EpochSlots(EpochSlotIndex, EpochSlots),
+    LowestSlot(LowestSlot),
```
Comment (mvines) [inline core/src/crds_value.rs]: But you said no more ABI changes last month!

This is fine I guess, we'll just be unable to do a rolling update from 1.0.x to 1.1.x next month.

Comment (aeyakovenko) [reply core/src/crds_value.rs]: Rolling update should still work.  Repair will be slow until the vast majority updates.  We don’t store the crds on disk.

Comment (aeyakovenko) [reply core/src/crds_value.rs]: @carllin wdyt?  should i keep the old fields in there and mark them as deprecated?  In theory the upgrade should allow both to exist, you just don't want to be in the minority.

Comment (aeyakovenko) [reply core/src/crds_value.rs]: the annoying thing is that it pulls in a big pile of crap, EpochIncompleteSlots, and all the compression stuff

**Pull request closed without merging**